### DISCUSSION TITLE
EDGETX BUG ，mixing cannot delete channels,

### DISCUSSION THREAD

**Discussion by wsnfly** [EdgeTX/edgetx#1523] [Q&A]
I wanted to test some functions of the system logic switch today, and set the test output on channels 31 and 32, but when I wanted to delete them, I found that they could not be deleted or replaced. Developers please take a look.

https://www.youtube.com/watch?v=bUFw3TmUSF0

**Comment by pfeerick** [comment]
Can you share the .yml file for that model thanks? I can't reproduce this behavior on 2.6... adding and deleting channels 31 and 32 is working for me. Also, FYI, as a bug report, this belongs in issues, not as a discussion Q&A question ;)

**Reply by wsnfly** [reply]
At the time I was setting up logic switches and mixers and wanted to implement some more complex logic. Suddenly all the channels of the mixing interface disappeared (but the output was still normal). At this time, I didn't think it was a bug, I thought it was my operation problem, and then I reset the mixing channels one by one. But 12, 31, 32 channels can no longer be deleted and replaced.
At the time I was using version 2.5.0. The problem still exists when I flash to version 2.6.0.
entire SD card file.

https://drive.google.com/file/d/1WL99J2piIHeTk87D7MifIV4wX2t2GJPc/view?usp=sharing

**Comment by Ktr128** [comment]
Checked on NV14 firmware 2.6. Everything works fine, removed. No response in the channel monitor.
